package.json

```diff
@@ -36,10 +36,9 @@
     "jsonlint:files": "find . -name \"*.json\" | grep -v -f .eslintignore",
     "lint": "eslint . --cache && echo \"eslint: no lint errors\"",
     "lint:fix": "eslint . --cache --fix && echo \"eslint: no lint errors\"",
-    "prettier": "yarn --silent prettier:files | xargs prettier --write",
-    "prettier:files": "find . -name \"*.js\" -o -name \"*.d.ts\" -o -name \"*.tsx\" | grep -v -f .eslintignore",
-    "prettier:check": "yarn --silent prettier:files | xargs prettier --list-different",
-    "prettier:ci": "prettier --list-different",
+    "prettier": "babel-node ./scripts/prettier.js write-changed",
+    "prettier:all":"babel-node ./scripts/prettier.js write",
+    "prettier:ci": "babel-node ./scripts/prettier.js",
```

Comment:
It would be great to explicit the default value: `check`.

Reply:
Actually, I think that we can remove this script command, remove the `CHANGED_FILES` env variable in the .circleci config file, simply run `yarn babel-node ./scripts/prettier.js check-changed`